torch/nn/modules/loss.py

```diff
@@ -1235,24 +1240,45 @@ class CTCLoss(_Loss):
 
     Inputs:
         log_probs: Tensor of size :math:`(T, N, C)` where `C = number of characters in alphabet including blank`,
-            `T = input length`, and `N = batch size`.
+        `T = input length`, and `N = batch size`.
             The logarithmized probabilities of the outputs
             (e.g. obtained with :func:`torch.nn.functional.log_softmax`).
-        targets: Tensor of size :math:`(N, S)` or `(sum(target_lengths))`.
-            Targets (cannot be blank). In the second form, the targets are assumed to be concatenated.
+        targets: Tensor of size :math:`(N, S)` or `(sum(target_lengths))`, where `N = batch size`, and
+        `S = maximum target length`.
+            Each element in the target sequence is a class index. Target index cannot be blank (default=0).
+            In the second form, the targets are assumed to be concatenated.
         input_lengths: Tuple or tensor of size :math:`(N)`.
-            Lengths of the inputs (must each be :math:`\leq T`)
+            Lengths of the inputs (must each be :math:`\leq T`). Lengths are specified for each sequence to achieve
+            masking. In effect, length is the stop index s for each target sequence,
+            such that :math: `target_n = targets[n,:s_n]` for each target in a batch.
         target_lengths: Tuple or tensor of size  :math:`(N)`.
-            Lengths of the targets
+            Lengths of the targets (may be :math: < S)
```

Comment:
Maybe *should be <= S if targets is a two-dimensional tensor* (but with `:math:`) would be clearer.